crates/fuel-core/src/graphql_api/ports.rs

```diff
@@ -64,7 +64,20 @@ use fuel_core_types::{
 };
 use std::sync::Arc;
 
-use super::storage::balances::TotalBalanceAmount;
+use super::storage::{
+    balances::TotalBalanceAmount,
+    coins::{
+        CoinsToSpendIndexKey,
+        IndexedCoinType,
+    },
+};
+
+pub struct CoinsToSpendIndexIter<'a> {
+    pub(crate) big_coins_iter:
+        BoxedIter<'a, Result<(CoinsToSpendIndexKey, IndexedCoinType), StorageError>>,
+    pub(crate) dust_coins_iter:
```

Comment:
In the future we plan to move GraphQL into its own service, so it will be nice either use `pub` or define a structure that can be used outside of the crate(like add constructor, getter, `into_inner` for example) =)

Reply:
I mad the fields `pub` in this commit: 253306085f404e17ae662f15c08a0d26caeaa4c1